app/presenters/date_question_presenter.rb

```diff
@@ -15,15 +13,24 @@ def response_label(value)
   end
 
   def selected_day
-    prefill_value_for(self, :day)
+    return if response_for_current_question.blank?
+    return response_for_current_question unless response_for_current_question.is_a? Hash
```

Comment:
Would we expect this to ever be valid and not be a hash? I wonder if this should just return nil if it's not a hash.